jbpm-wb-common/jbpm-wb-common-api/src/main/java/org/jbpm/workbench/common/preferences/ManagePreferences.java

```diff
@@ -27,16 +27,36 @@
 @WorkbenchPreference(identifier = "ManagePreferences", bundleKey = "ManagePreferences.Label")
 public class ManagePreferences implements BasePreference<ManagePreferences> {
 
+    public static String COMPLETED_COLOR = "#C0C0C0";
+    public static String COMPLETED_BORDER_COLOR = "#030303";
+    public static String ACTIVE_BORDER_COLOR = "#FF0000";
+
     public static List<Integer> PAGINATION_OPTIONS = Arrays.asList(10, 20, 50, 100);
     public static Integer DEFAULT_PAGINATION_OPTION = 10;
 
     @Property(bundleKey = "ManagePreferences.ItemsPerPage", helpBundleKey = "ManagePreferences.ItemsPerPage.Help",
             formType = PropertyFormType.NATURAL_NUMBER, validators = ItemPerPageValidator.class)
     private Integer itemsPerPage;
 
+    @Property(bundleKey = "ManagePreferences.ProcessInstanceDiagramCompletedNodeColor", helpBundleKey = "ManagePreferences.ProcessInstanceDiagramCompletedNodeColor.Help",
+            formType = PropertyFormType.COLOR)
+    private String processInstanceDiagramCompletedNodeColor;
```

Comment:
Wouldn't be a different representation than String be more suitable? It would provide oob validation of the string format. Or is this achieved differently?

Reply:
@jstastny-cz Here we are using this validation https://github.com/kiegroup/appformer/blob/f8f3737a854dc64f1075b076514c8630bf3da5f2/uberfire-extensions/uberfire-widgets/uberfire-widgets-properties-editor/uberfire-widgets-properties-editor-api/src/main/java/org/uberfire/ext/properties/editor/model/validators/ColorValidator.java that is embedded in the PropertyEditorColorPicker.